tutorial/src/Constants.lagda

```diff
@@ -1,13 +1,13 @@
 % Constant Definitions
 
-Import List
+Import list
 ===========
 
 \begin{code}
 module Constants where
 
-open import Data.Bool using (Bool; true; false)
-open import Data.Nat using (ℕ; zero; suc)
+open import Sets.Enumerated using (Bool; true; false)
+open import Sets.Recursive using (ℕ; zero; suc)
```

Comment:
I've tried to prefer importing from the standard library wherever possible, using Data.Nat rather than Sets.Recursive and similarly prefering Data.Bool over Sets.Enumerated, in order to make the examples as standalone as possible and also to encourage the use of the standard library. Are there particular reasons not to use the standard library in this case?